kit/scrolls/src/wrapScrollableComponent/wrapScrollableComponent.ios.tsx

```diff
@@ -58,7 +69,15 @@ export function wrapScrollableComponent<Props extends ScrollViewProps>(
                     scrollEventThrottle={16}
                     onLayout={horizontal ? undefined : onLayout}
                     onContentSizeChange={horizontal ? undefined : onContentSizeChange}
+                    contentInset={automaticInsets ? undefined : contentInset}
```

Comment:
I believe that you should also disable `scrollIndicatorInsets` here: https://reactnative.dev/docs/scrollview#scrollindicatorinsets-ios